package.json

```diff
@@ -41,8 +41,8 @@
     "docs": "docsify serve docs",
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js --inject-globals false docker.test.js utils.test.js display.test.js",
     "watch": "node --experimental-vm-modules node_modules/jest/bin/jest.js --watch --inject-globals false",
-    "lint": "eslint *.js *.test.js bin/*.js",
-    "pretty": "prettier --write *.js data/*.json bin/*.js --trailing-comma=none"
+    "lint": "eslint *.js *.cjs *.test.js bin/*.js",
```

Comment:
Should we lint .cjs files? It seems to be flagging many issues.

Reply:
Reverted